### PR TITLE
Added jQuery, MySQL, and Adobe Photoshop

### LINKED ISSUE #567: Icon Requests (jQuery, Photoshop, PHP, MySQL)
I would like to request the following brand icons:

- jQuery
- Photoshop
- PHP
- MySQL

I actually have a jQuery SVG that can be used. [jQuery.svg.zip](https://github.com/simple-icons/simple-icons/files/1330160/jQuery.svg.zip)

I don't have resources for the other icons but would really like them. The typical MySQL logo is rectangular, so I would recommend the [logo from Planet MySQL](https://planet.mysql.com/images/planet-logo.svg).

### PR DESCRIPTION
**Pull request by xdesro**
Closes #567 . Let me know if any issues! Thanks so much, gang.

### CODE REVIEW

**Comment by ericcornelissen** [comment]
I know it is not listed on the [logos page](https://brand.jquery.org/logos/#incorrect-logo-usage) of jQuery, but shouldn't we use the small logo they use as favicon? :thinking:

**Comment by xdesro** [comment]
I'm happy to swap it, @ericcornelissen -- that makes more sense to me for what our purposes are. I just didn't know how much we're comfortable straying from their brand guidelines. Also, would we still use the primary "black" #333333 they indicate [here](http://brand.jquery.org/colors/#jquery-black), or use the blue (because the symbol is blue)?

**Comment by birjolaxew** [comment]
@xdesro The color should be the "brand color" - that is, the color people recognize the brand by. In this case that is "jQuery Primary Blue" (`#0769AD`). If multiple such colors exists (as in this case), assume a light background.

I agree with @ericcornelissen tho, it's fine to use the logomark instead of the whole logo.

**Comment by xdesro** [comment]
Updated!

File: icons/adobephotoshop.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg aria-labelledby="simpleicons-adobephotoshop-icon" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><title id="simpleicons-adobephotoshop-icon">jQuery icon</title><path d="M.5.8v22.4h23V.8H.5zm1 .9h21.1v20.5H1.5V1.7zM6.1 6c.1-.1.2-.1.2-.1H8.7c1.3 0 2.3.3 2.8 1s.9 1.4.9 2.4c0 1.2-.4 2.1-1.1 2.7s-1.7.8-2.8.8h-.7V16.6H6.2h-.1V6zm1.6 5.1h.7c.6 0 1.1-.1 1.5-.4.4-.3.7-.8.7-1.5 0-.6-.2-1.1-.5-1.4-.3-.2-.8-.4-1.4-.4h-.9v3.7h-.1zM16 10c-.4 0-.6.1-.8.2-.1.2-.2.3-.2.5s.1.3.2.5.5.4 1 .6c.8.4 1.3.7 1.6 1.1.3.4.4.8.4 1.3 0 .8-.3 1.3-.8 1.7s-1.2.6-2 .6c-.4 0-.8 0-1.1-.1-.3-.1-.6-.2-.8-.3 0 0-.1 0-.1-.1v-1.5-.1h.1c.3.2.6.4.9.4s.7.1 1 .1c.4 0 .7-.1.8-.2.2-.1.3-.3.3-.5s-.1-.3-.2-.5c-.2-.2-.5-.4-1-.6-.7-.3-1.2-.7-1.5-1.1-.3-.4-.4-.8-.4-1.4 0-.6.2-1.1.7-1.6s1.1-.7 1.9-.7c.4 0 .8 0 1.1.1.3.1.5.1.7.2l.1.1V10.2h-.1c-.2-.1-.5-.2-.8-.3-.4.1-.7.1-1 .1z"/></svg>
```
Comment (birjolaxew) [inline icons/adobephotoshop.svg]: `<title>jQuery icon</title>` should be changed to Adobe Photoshop

Comment (xdesro) [reply icons/adobephotoshop.svg]: I'm an idiot

**Comment by birjolaxew** [comment]
:+1: Thanks for the contribution! Real quick work too!

**Pull request merged**